locales/de.json

```diff
@@ -819,12 +746,14 @@
 		},
 		"button": {
 			"createCourse": "Erstelle einen neuen Kurs",
-			"createFirstCourse": "Erstelle deinen ersten Kurs",
```

Comment:
This still seems to be used in the code

Reply:
You are right, changed used key, fixed ;)